specs/logging_implementation.adoc

```diff
@@ -0,0 +1,174 @@
+= Specification for logging implementation
+
+Logs and alerts need to be treated as messages emitted by a component.
+
+== Problem description
+
+* Logs and alerts need to be treated as messages emitted by a component.
```

Comment:
Log nothing but a information, erros. etc. It helps  an end_user for the debugging the problem. Alert also a message but it is immediately forward to end_user. Alerts can be debug with the help of log message.